flink-runtime/src/main/java/org/apache/flink/runtime/metrics/MetricNames.java

```diff
@@ -57,7 +57,11 @@ private MetricNames() {}
     public static final String MEMORY_COMMITTED = "Committed";
     public static final String MEMORY_MAX = "Max";
 
-    public static final String IS_BACKPRESSURED = "isBackPressured";
+    public static final String IS_BACK_PRESSURED = "isBackPressured";
+    public static final String IS_BACK_PRESSURED_RATIO = "isBackPressuredRatio";
```

Comment:
I see that `IS_BACK_PRESSURED_RATIO` name is derived from `IS_BACK_PRESSURED` but it still sounds strange to me. Looking at just a single metric is even more confusing.
So I'd name it  `BACK_PRESSURED_RATIO` (also in docs).

ditto: `IS_CAUSING_BACK_PRESSURE_RATIO`